src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -170,6 +171,7 @@ export const Checkout: React.FC<CheckoutProps> = ({
   const [signUiState, setSignUiState] = useState<SignUiState>('NOT_STARTED')
   const [emailUpdateLoading, setEmailUpdateLoading] = useState(false)
   const [ssnUpdateLoading, setSsnUpdateLoading] = useState(false)
+  const [showFailModal, setShowFailModal] = useState(false)
```

Comment:
I think we should stick to naming boolean variables with a prefix of `is` or `has` (or in special cases `should` or `can`). So this could be `isShowingFailModal` or `isFailModalVisible` for example 🙏